connectors/analytics/conviva/src/main/java/com/theoplayer/android/connector/analytics/conviva/ads/AdReporter.kt

```diff
@@ -226,11 +263,20 @@ class AdReporter(
             // - `c3.csid`: the content’s sessionID;
             // - `contentAssetName`: the content's assetName.
             val contentAssetName = convivaHandler.contentAssetName
-            val adMetadata = collectAdMetadata(ad) + mapOf(
+            val adTechnology =
+                if (ad.integration == AdIntegrationKind.THEO_ADS) "Server Guided" else calculateAdTypeAsString(
+                    ad
+                )
```

Comment:
Couldn't we put the "Server Guided" check into `calculateAdTypeAsString`?

Reply:
Indeed would be better, will update it